amoro-common/src/main/java/org/apache/amoro/hive/AuthenticatedHiveClientPool.java

```diff
@@ -65,6 +65,7 @@ public static HMSClient createHiveMetaStoreClient(HiveConf hiveConf) {
     } catch (MetaException e) {
       throw new RuntimeException("Failed to connect to Hive Metastore", e);
     } catch (Throwable t) {
+      t.printStackTrace();
```

Comment:
```suggestion
      LOG.error( "Failed to call createHiveMetaStoreClient", t);
```

Reply:
I don't see this print introduced by this PR?